stride/assetlist.json

```diff
@@ -598,6 +598,42 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/celestia/images/celestia.svg"
         }
       ]
+    },
+    {
+      "denom_units": [
+        {
+          "denom": "staISLM",
+          "exponent": 0
+        },
+        {
+          "denom": "stISLM",
+          "exponent": 18
+        }
+      ],
+      "base": "staISLM",
+      "name": "Stride Staked ISLM",
+      "display": "stISLM",
+      "symbol": "stISLM",
+      "traces": [
+        {
+          "type": "liquid-stake",
+          "counterparty": {
+            "chain_name": "haqq",
+            "base_denom": "aISLM"
+          },
+          "provider": "Stride"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/islm.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/islm.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/stride/images/islm.png",
```

Comment:
This would be the wrong image to use. The islm image already exists under haqq, so we don't need the duplication. But also the stISLM has a different image